pkg/tiles/feature/feature.go

```diff
@@ -2,7 +2,7 @@ package feature
 
 import connection "github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/connection"
 
-type FeatureType int64
+type FeatureType int64 //nolint:revive // name type would be too ambigous
```

Comment:
Currently this identifier will look like this feature.FeatureType, so there is no need to stutter and we can simplify it to feature.Type

Reply:
changed